storybook/stories/core/components/Input/SelectInput.stories.js

```diff
@@ -346,6 +346,14 @@ CustomTemplate.parameters = {
   },
 };
 
+export const TranslateOptions = Template.bind({});
```

Comment:
no need to add a new story here. in the top introduce the `translateOptions` prop (see line 10 above). this way devs referring the storybook can toggle it and as well understand the meaning.